### DISCUSSION TITLE
Is it possible to set "Lock aspect ratio" to all figures in a word render?

### DISCUSSION THREAD

**Discussion by jeanvincentlebe** [quarto-dev/quarto-cli#11614] [Q&A]
### Description

I would like to set by default "lock aspect ratio" to all figures in a word document rendered. This would facilitate to keep the good layout of the plots.
Thanks for your help :) 

Quarto version: 1.4.550
Windows 10

**Comment by mcanouil** [comment]
First, you should upgrade (if you can) your version of Quarto which is starting to be very old.
Current is 1.6.39.

I don't know what you mean by "lock aspect ratio" or what is a "good layout of the plots".
Quarto does not do resizing; it uses the images size and for computed one uses some default you can change.

- https://quarto.org/docs/computations/execution-options.html#figure-options

For more help, please provide a reproducible example.
The goal is to make it as easy as possible for us to recreate your problem/use case so that we can fix it or help you: please help us help you! Thanks.

---

You can share a **self-contained "working" (reproducible)** Quarto document using the following syntax, _i.e._, using more backticks than you have in your document (usually four ` ```` `).
See <https://quarto.org/bug-reports.html#small-is-beautiful-aim-for-a-single-document-with-10-lines>.

If you have multiple files (and if it is **absolutely required** to have multiple files), please share as a Git repository.

<table>
<tr><th>R</th><th>Python</th></tr>
<tr><td>

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
engine: knitr
---

This is a reproducible Quarto document.

{{< lipsum 1 >}}

```{r}
x <- c(1, 2, 3, 4, 5)
y <- c(1, 4, 9, 16, 25)

plot(x, y)
```

![An image]({{< placeholder 600 400 >}}){#fig-placeholder}

{{< lipsum 1 >}}

The end after @fig-placeholder.
````
`````

</td><td>

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
engine: jupyter
---

This is a reproducible Quarto document.

{{< lipsum 1 >}}

```{python}
import matplotlib.pyplot as plt

x = [1, 2, 3, 4, 5]
y = [1, 4, 9, 16, 25]

plt.plot(x, y)
plt.show()
```

![An image]({{< placeholder 600 400 >}}){#fig-placeholder}

{{< lipsum 1 >}}

The end after @fig-placeholder.
````
`````

</td></tr>
</table>

Additionally and if not already given, please share the output of `quarto check` within a code blocks (*i.e.*, using three backticks ` ```txt `), see <https://quarto.org/bug-reports.html#check>.

**Reply by jeanvincentlebe** [reply]
The "Lock aspect ratio" is a setting in Word: "Picture format" -> "size", then:
![ImageSizeWord](https://github.com/user-attachments/assets/1f9497f4-b84d-44f7-b835-898e56c14d89)

By "good layout of the plots" I meant how the plots come out of Quarto with good aspect ratio. It can be deformed in word later on if not set properly.

It is not a bug, I was just wandering if this is possible to set as default in the generated word document. It would be convenient especially when many figures are generated.

I'll try to come up with a reproducible example.

**Reply by mcanouil** [reply]
You can try setting this in a reference document as for the styles: https://quarto.org/docs/output-formats/ms-word-templates.html

If it does not work, you would need to go for a more advanced solution such as post-processing the produced file to change the setting.

Note that the template system is coming straight from Pandoc which has some limitations.
